{"name": "test_1_login_error[error_phone0]", "status": "passed", "parameters": [{"name": "error_phone", "value": "{'phone': '', 'pwd': 'python', 'expect': '请输入手机号'}"}], "start": 1568898307602, "stop": 1568898309072, "uuid": "0894f43c-7154-4325-861e-9695ab42376d", "historyId": "1f7f5f9e7b108ec3dcac4bac9d268d98", "testCaseId": "e8c1f96b3a1ee5611ba6f39770dbf5d5", "fullName": "cases.test_login.TestLogin#test_1_login_error", "labels": [{"name": "tag", "value": "login"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "zhzhuwang"}, {"name": "thread", "value": "19908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}